lib/Qi/Controller/Assets.pm

```diff
@@ -0,0 +1,44 @@
+package Qi::Controller::Assets;
+use Mojo::Base 'Mojolicious::Controller';
+use Qi::Model::Base;
+use SQL::Abstract;
+
+my $table = 'assets';
+
+sub get {
+  my $self = shift;
+  my $response = $self->req->json;
+  $self->stash('asset' => $self->base->select($response , $table));
```

Comment:
This will eventually be $self->assets->select(...)

And with the `has table` above, instead of $table, it's `$self->table`.